backend/handler/tests/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import alembic.config
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from config.config_loader import ConfigLoader
+from models.platform import Platform
+from models.rom import Rom
+
+engine = create_engine(ConfigLoader.get_db_engine(), pool_pre_ping=True)
+session = sessionmaker(bind=engine, expire_on_commit=False)
+
+
+@pytest.fixture(scope="session")
```

Comment:
We still need migration because the `main.py` migration is not executed when running tests, because `maun.py` is not executed.